src/apis/constants.ts

```diff
@@ -7,9 +7,7 @@ import { WeightToFeeCoefficient } from "@polkadot/types/interfaces/support";
 
 export interface ConstantsAPI {
     getDotExistentialDeposit(): Balance;
-    getIssuePeriod(): BlockNumber;
     getPolkaBtcExistentialDeposit(): Balance;
-    getReplacePeriod(): BlockNumber;
```

Comment:
Are these constants no longer part of the parachain? I didn't have the most up-to-date version when adding this API, because of the compilation time

Reply:
They're now storage fields so they can be actively updated.